logging/google/cloud/logging/handlers/__init__.py

```diff
@@ -15,4 +15,6 @@
 """Python :mod:`logging` handlers for Google Cloud Logging."""
 
 from google.cloud.logging.handlers.handlers import CloudLoggingHandler
+from google.cloud.logging.handlers.GAEHandler import GAEHandler
```

Comment:
I think  `handlers.GAEHandler` and `handlers.GKEHanlder` imports go above the handlers.handlers imports.